values.yaml

```diff
@@ -125,6 +129,9 @@ extraEnv: {}
 extraArgs: {}
 extraVolumes: {}
 
+annotations: {}
+imagePullSecrets: ""
```

Comment:
Can you provide a little more context on `imagePullSecrets`? Is this required for lint'ing or?

Reply:
Same here it is required for lint'ing.